aio/content/examples/getting-started/src/app/app.module.ts

```diff
@@ -39,20 +39,21 @@ import { ShippingComponent } from './shipping/shipping.component';
     ])
   ],
   // #enddocregion product-details-route, cart-route
-  // #docregion declare-product-alerts
+  // #docregion declare-product-alerts, product-details-route
```

Comment:
Ending the `product-details-route` region and then immediately restarting it is redundant. Instead just remove both references here:

```suggestion
  // #enddocregion cart-route
  // #docregion declare-product-alerts
```

Reply:
Thanks. has updated.